packages/github-action/action.yml

```diff
@@ -12,6 +12,39 @@ inputs:
 runs:
   using: "composite"
   steps:
+    - name: Add initial reaction and create progress comment
```

Comment:
would it easier to wrap these logic in typescript, or use `gh` command line, rather than invoking api by your self?